deps/ledger-zxlib/app/common/view.c

```diff
@@ -33,28 +33,30 @@
 
 #include <string.h>
 #include <stdio.h>
+#include <stdbool.h>
 
 view_t viewdata;
 
-void h_approve(unsigned int _) {
-    UNUSED(_);
```

Comment:
What did this command do?

Reply:
There are two changes here UNUSED-> __Z_UNUSED and view_idle_show second argument. So no real change here. These are internals of zxlib. I am not too familiar with zxlib's internal details.